pkg/auth/providers/oidc/oidc_provider.go

```diff
@@ -221,12 +231,17 @@ func (o *OpenIDCProvider) RefetchGroupPrincipals(principalID string, secret stri
 		logrus.Errorf("[generic oidc] refetchGroupPrincipals: error getting user by principalID: %v", err)
 		return groupPrincipals, err
 	}
+	var oauthToken oauth2.Token
+	if err := json.Unmarshal([]byte(secret), &oauthToken); err != nil {
+		return nil, err
+	}
+
 	//do not need userInfo or oauth2Token since we are only processing groups
```

Comment:
I don't think this comment is relevant anymore

Reply:
removed